react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is React?
+
+React is a javascript library that allows developers to easily build out large scale and complex projects by breaking up functionality into reusable components
+
+- What is a React component?
+
+A react component is a collection of all the necessary elements (HTML, CSS and Javascript) to build out one piece of functionality
+
+- How do you mount a React app (root component) to the DOM?
+
+Using the export default keyword
```

Comment:
Not quite. Take a look in `main.tsx`. How are we attaching the react app to the DOM?